bitnami/minio/values-production.yaml

```diff
@@ -143,9 +143,14 @@ statefulset:
 ##
 # existingSecret:
 
-## Mount MinIO secret as a file instead of passing environment variable
+## Mount MinIO secret as a file instead of passing environment variable. If useSecretValues is true it will mount the values from the minio secret. Otherwise a local file can be used.
 ##
-useCredentialsFile: false
+credentialsFile:
+  enabled: false
+  useSecretValues: true
+  mountPath: /opt/bitnami/minio/secrets/
+  accessKeyFile: access-key
+  secretKeyFile: secret-key
```

Comment:
Could you extend the description of each parameter a little bit more? Check the rest of the file for examples of the format. You can just use the descriptions from the README.

Reply:
I copy pasted the README descriptions into comments above the parameters with formatting like the rest of the file (in both values.yaml and values-production.yaml)